CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+
+-    Set PayPal Standard payment gateway id to original (#5414)
```

Comment:
```suggestion
-    Restore PayPal Standard functionality that was affected by a name change in GiveWP 2.9.0 (#5414)
```

@JasonTheAdams This is the language I'm recommending for the `readme.txt` changelog as well.